### PR TITLE
Tailwind width patch

### PR DESCRIPTION
**Pull request by abinashpa**
## Summary
/report page's UI is breaking because 2 tailwind values are not present this PR will fix those issues.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by abinashpa** [comment]
Screenshot 2022-07-13 at 12 12 34 PM

File: app/javascript/src/components/Reports/reportList/reportCard.tsx [hunk 1]
```diff
@@ -8,8 +8,8 @@ const ReportCard = ({
   url
 }) => (
   <NavLink end to={`/reports/${url}`}>
-    <div className="w-512 h-[148px] rounded-lg	border-2	border-miru-gray-100 box-border	p-5 mt-5 flex hover:shadow-c1  cursor-pointer">
-      <div className="w-120 h-120 bg-miru-gray-100 flex justify-center items-center rounded	">
+    <div className="w-128 h-[148px] rounded-lg	border-2	border-miru-gray-100 box-border	p-5 mt-5 flex hover:shadow-c1  cursor-pointer">
```
Comment (onkar-saeloun) [inline app/javascript/src/components/Reports/reportList/reportCard.tsx:11]: do not change anything in component. Just add changes to tailwind conf. file

File: tailwind.config.js [hunk 2]
```diff
@@ -42,6 +42,8 @@ module.exports = {
       },
       width: {
         fit: "fit-content",
+        30: "7.5rem", //120px
+        128: "32rem", // 512px
```
Comment (onkar-saeloun) [inline tailwind.config.js:46]: Add this,
120: "120px",
 512: "512px",
 Instead of above

File: tailwind.config.js [hunk 3]
```diff
@@ -42,6 +42,8 @@ module.exports = {
...
       },
       height: {
         120: "120px",
```
Comment (onkar-saeloun) [inline tailwind.config.js:49]: You did overrided in height as well. Please revert the change if you haven't intentionally did anything. 
Screenshot 2022-07-13 at 11 45 07 AM

**Review by onkar-saeloun — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 3 above)
Comment (abinashpa) [reply tailwind.config.js:49]: I didn't override any height value.

File: tailwind.config.js (hunk 2 above)
Comment (abinashpa) [reply tailwind.config.js:46]: That's breaking the tailwind convention.

**Review by onkar-saeloun — APPROVED** [review]
LGTM
